examples/ES/README.md

```diff
@@ -20,18 +20,14 @@ TODO
 
 ### Distributed Training
 
-#### Learner
-```sh
-python learner.py 
-```
+To replicate the performance reported above, we encourage training with 96 CPUs. 
+If you haven't created a cluster before, enter the following command to create a cluster with 96 CPUs. For more information about the cluster, please refer to our [documentation](https://parl.readthedocs.io/en/latest/parallel_training/setup.html).
```

Comment:
should we add `python learner.py`?

Reply:
Added. Note I have renamed `learner.py` to `train.py`